api/graphql/schema.graphql

```diff
@@ -747,6 +760,26 @@ type Query {
   pools: [Pool]!
     @all(scopes: ["authorizedToView", "notArchived"])
     @canResolved(ability: "view")
+    @deprecated(reason: "pools is deprecated. Use poolsPaginated instead.")
+  poolsPaginated(
+    includeIds: [UUID!] @in(key: "id")
+    excludeIds: [UUID!] @notIn(key: "id")
+    where: PoolFilterInput
+    orderByTeamDisplayName: PoolTeamDisplayNameOrderByInput @scope
+    orderBy: _
+      @orderBy(
+        relations: [
+          { relation: "user", columns: ["first_name", "email"] }
+          { relation: "classification", columns: ["group", "level"] }
+        ]
+      )
+  ): [Pool]!
+    @paginate(
+      defaultCount: 10
+      maxCount: 1000
+      scopes: ["authorizedToView", "notArchived"]
```

Comment:
Working on the tests and was like "why is the archived scope failing?!" Then I remembered this...

Do we want this? :thinking: Or should I just not consider archived in the tests? @tristan-orourke

Reply:
We can take off the built-in notArchived scope, though most places querying pools will want to add it. Could we make it enabled by default?

Reply:
Hmm, I'll take a look. I'm pretty sure you can provide defaults and then when you pass something in, it overrides it :thinking:

Reply:
I don't think there's an easy way to make it default. That's fine.